internal/pkg/template/templates/environment/cf.yml

```
@@ -356,6 +356,9 @@ Resources:
       LoadBalancerArn: !Ref PublicLoadBalancer
       Port: 443
       Protocol: HTTPS
+{{ if .PublicHTTPConfig.SSLPolicy }}
+      SslPolicy: {{ .PublicHTTPConfig.SSLPolicy }}
```

nit (here and below, and above in integ tests):
```suggestion
      SSLPolicy: {{ .PublicHTTPConfig.SSLPolicy }}
```